ingestion-beam/src/main/java/com/mozilla/telemetry/transforms/PubsubMessageToTableRow.java

```diff
@@ -139,6 +149,14 @@ private PubsubMessageToTableRow(ValueProvider<String> tableSpecTemplate,
 
     final TableDestination tableDestination = new TableDestination(tableSpec, null,
         TIME_PARTITIONING);
+
+    if (tableRowFormat != null && tableRowFormat.get() == TableRowFormat.raw) {
```

Comment:
```suggestion
    if (tableRowFormat.get() == TableRowFormat.raw) {
```

We should be guaranteed this is non-null after the check in the constructor, right?

Reply:
turns out I wasn't setting the default properly in `SinkOptions`. I fixed that and we no longer need to check for null here or in the constructor.